ui-v2/tests/acceptance/dc/nodes/index.feature

```diff
@@ -16,40 +16,6 @@ Feature: dc / nodes / index
     Then the url should be /dc-1/nodes
     And the title should be "Nodes - Consul"
     Then I see 3 node models
-  Scenario: Seeing the leader in unhealthy listing
-    Given 3 node models from yaml
-    ---
-      - Address: 211.245.86.75
-        Checks:
-          - Status: warning
-            Name: Warning check
-      - Address: 10.0.0.1
-      - Address: 10.0.0.3
-    ---
-    When I visit the nodes page for yaml
-    ---
-      dc: dc-1
-    ---
-    Then the url should be /dc-1/nodes
-    Then I see 3 node models
-    And I see leader on the unHealthyNodes
-  Scenario: Seeing the leader in healthy listing
-    Given 3 node models from yaml
-    ---
-      - Address: 211.245.86.75
-        Checks:
-          - Status: passing
-            Name: Passing check
-      - Address: 10.0.0.1
-      - Address: 10.0.0.3
-    ---
-    When I visit the nodes page for yaml
-    ---
-      dc: dc-1
-    ---
-    Then the url should be /dc-1/nodes
-    Then I see 3 node models
-    And I see leader on the healthyNodes
   Scenario: Searching the nodes with name and IP address
```

Comment:
Can we rewrite these tests so we keep a test for showing the leader?

Reply:
Added back. 😓  I dint' realize the only leader tests were tucked in these.